assembly/src/parsers/ast/serde/deserialization.rs

```diff
@@ -5,6 +5,8 @@ use super::{
 use crate::errors::SerializationError;
 use vm_core::{utils::collections::Vec, utils::string::String, Felt};
 
+const PROC_DIGEST_SIZE: usize = 24;
```

Comment:
We should probably move this to a higher level so that we could use it in `ast/context.rs` as well.